run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -x
+set -e
+[ -n "${CLEAN_CONFIG:-}" ] && rsync -a --delete /app/config-orig/ /app/config/
+[ -f "/app/config/global.php" ] || rsync -a --delete /app/config-orig/ /app/config/
+chown -R www-data:www-data /app/config
+/etc/init.d/apache2 start
```

Comment:
It looks like the upstream image launches apache slightly differently:

https://github.com/tutumcloud/tutum-docker-php/blob/master/run.sh#L3-5